tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -609,9 +609,10 @@ def test_add_fluorescence_traces(self):
             self.neuropil_roi_response_series_metadata["unit"],
         )
 
-        self.assertEqual(
+        self.assertAlmostEqual(
             fluorescence["Neuropil"].rate,
             self.segmentation_extractor.get_sampling_frequency(),
+            places=3,
         )
```

Comment:
You can actually try returning this to `assertEqual` and pinning to https://github.com/catalystneuro/roiextractors/pull/212, which should fix it